packages/patterns/components/c-link/c-link.twig

```diff
@@ -1,3 +1,3 @@
-<a class="c-link {{ modifier_class }} {{ c_link_classes }}" href="{{ c_link_url }}">
+<a class="c-link {{ modifier_class }} // {{ c_link_classes }}" href="{{ c_link_url }}">
```

Comment:
Is this commented out?

Reply:
No - that's a text separator for between the actual pattern's name and utilities, though it does look like a comment which I hadn't thought of, and now that I'm looking at it, I think that should only be on modules. It's intended to help with readability in the DOM e.g. 
![Screen Shot 2019-07-30 at 5 39 29 PM](https://user-images.githubusercontent.com/445861/62167395-2179be80-b2f1-11e9-87c7-b74ab6729707.png)